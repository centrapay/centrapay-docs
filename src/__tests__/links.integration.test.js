/* eslint-disable complexity */
const { listSiteLinks } = require('../utils/listSiteLinks');

test('the build should not break any links', () => {
  const uniqueLinks = listSiteLinks('_site');
  const sortedLinks = Array.from(uniqueLinks).sort();
  expect(sortedLinks).toMatchSnapshot();
});

