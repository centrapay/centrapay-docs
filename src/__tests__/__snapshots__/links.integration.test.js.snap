// Vitest Snapshot v1

exports[`the build should not break any links 1`] = `
[
  "",
  "/404",
  "/accounts",
  "/api",
  "/api/account-memberships",
  "/api/account-memberships#account-member",
  "/api/account-memberships#account-memberships",
  "/api/account-memberships#add-member-experimental",
  "/api/account-memberships#contents",
  "/api/account-memberships#list-account-memberships-for-authenticated-subject",
  "/api/account-memberships#list-account-memberships-for-specific-user",
  "/api/account-memberships#list-members",
  "/api/account-memberships#models",
  "/api/account-memberships#operations",
  "/api/account-memberships#revoke-member",
  "/api/account-memberships#top",
  "/api/accounts",
  "/api/accounts#account",
  "/api/accounts#accounts",
  "/api/accounts#contents",
  "/api/accounts#create-an-account",
  "/api/accounts#get-an-account",
  "/api/accounts#models",
  "/api/accounts#operations",
  "/api/accounts#subscription",
  "/api/accounts#top",
  "/api/accounts#update-an-account",
  "/api/accounts#update-subscriptions",
  "/api/api-keys",
  "/api/api-keys#api-key",
  "/api/api-keys#api-keys",
  "/api/api-keys#contents",
  "/api/api-keys#create-an-api-key",
  "/api/api-keys#list-api-keys",
  "/api/api-keys#models",
  "/api/api-keys#operations",
  "/api/api-keys#top",
  "/api/api-keys#update-an-api-key",
  "/api/asset-categories",
  "/api/asset-transfers",
  "/api/asset-transfers#asset-transfer-lifecycle",
  "/api/asset-transfers#asset-transfers",
  "/api/asset-transfers#contents",
  "/api/asset-transfers#create-an-asset-transfer",
  "/api/asset-transfers#get-an-asset-transfer",
  "/api/asset-transfers#list-asset-transfers-experimental",
  "/api/asset-transfers#resolve-claimable-assets-experimental",
  "/api/asset-transfers#top",
  "/api/asset-types",
  "/api/asset-types#supported-asset-types",
  "/api/asset-types#top",
  "/api/assets",
  "/api/assets#archive-asset",
  "/api/assets#asset",
  "/api/assets#assets",
  "/api/assets#contents",
  "/api/assets#get-asset",
  "/api/assets#gift-cards",
  "/api/assets#list-assets-for-account",
  "/api/assets#listing-asset-transactions-experimental",
  "/api/assets#models",
  "/api/assets#money",
  "/api/assets#operations",
  "/api/assets#tokens-experimental",
  "/api/assets#top",
  "/api/auth",
  "/api/auth#account-flags",
  "/api/auth#api-keys",
  "/api/auth#auth",
  "/api/auth#authenticate-org-account-with-user-access-token",
  "/api/auth#authenticate-with-api-key",
  "/api/auth#authenticate-with-user-access-token",
  "/api/auth#authenticating-api-calls",
  "/api/auth#claims",
  "/api/auth#contents",
  "/api/auth#permissions",
  "/api/auth#roles-and-permissions",
  "/api/auth#top",
  "/api/auth#user-access-tokens",
  "/api/bank-account-approvals",
  "/api/bank-account-approvals#accept-a-bank-account-approval-experimental",
  "/api/bank-account-approvals#bank-account-approval",
  "/api/bank-account-approvals#bank-account-approval-activity",
  "/api/bank-account-approvals#bank-account-approval-types",
  "/api/bank-account-approvals#bank-account-approvals",
  "/api/bank-account-approvals#contents",
  "/api/bank-account-approvals#decline-a-bank-account-approval-experimental",
  "/api/bank-account-approvals#get-bank-account-approval-experimental",
  "/api/bank-account-approvals#list-bank-account-approvals-experimental",
  "/api/bank-account-approvals#models",
  "/api/bank-account-approvals#operations",
  "/api/bank-account-approvals#request-bank-account-approval-experimental",
  "/api/bank-account-approvals#top",
  "/api/bank-account-connection-intents",
  "/api/bank-account-connection-intents#authorize-bank-account-connection-intent-experimental",
  "/api/bank-account-connection-intents#bank-account-connection-intent",
  "/api/bank-account-connection-intents#bank-account-connection-intent-types",
  "/api/bank-account-connection-intents#bank-account-connection-intents",
  "/api/bank-account-connection-intents#contents",
  "/api/bank-account-connection-intents#create-bank-account-connection-intent-experimental",
  "/api/bank-account-connection-intents#models",
  "/api/bank-account-connection-intents#operations",
  "/api/bank-account-connection-intents#top",
  "/api/bank-accounts",
  "/api/bank-accounts#adding-a-direct-debit-authority-to-a-bank-account",
  "/api/bank-accounts#bank-account",
  "/api/bank-accounts#bank-account-approval-type-summary-experimental",
  "/api/bank-accounts#bank-account-balance-experimental",
  "/api/bank-accounts#bank-account-create",
  "/api/bank-accounts#bank-account-list",
  "/api/bank-accounts#bank-account-type-experimental",
  "/api/bank-accounts#bank-accounts",
  "/api/bank-accounts#bank-authority-get",
  "/api/bank-accounts#contents",
  "/api/bank-accounts#creating-a-bank-account",
  "/api/bank-accounts#creating-a-bank-authority-deprecated",
  "/api/bank-accounts#get-bank-account-balance-experimental",
  "/api/bank-accounts#get-information-about-a-bank-account",
  "/api/bank-accounts#get-information-about-a-bank-authority-deprecated",
  "/api/bank-accounts#list-bank-accounts",
  "/api/bank-accounts#list-bank-authorities-deprecated",
  "/api/bank-accounts#models",
  "/api/bank-accounts#operations",
  "/api/bank-accounts#top",
  "/api/bank-accounts#verify-a-bank-account",
  "/api/bank-accounts#verify-a-bank-authority-deprecated",
  "/api/bank-accounts#verify-bank-account",
  "/api/batch-types/farmlands",
  "/api/batch-types/farmlands-external-asset",
  "/api/batch-types/farmlands-external-asset#account",
  "/api/batch-types/farmlands-external-asset#card",
  "/api/batch-types/farmlands-external-asset#contact",
  "/api/batch-types/farmlands-external-asset#contents",
  "/api/batch-types/farmlands-external-asset#example",
  "/api/batch-types/farmlands-external-asset#example-jsonl-file",
  "/api/batch-types/farmlands-external-asset#example-models",
  "/api/batch-types/farmlands-external-asset#farmlands-external-asset-batch",
  "/api/batch-types/farmlands-external-asset#models",
  "/api/batch-types/farmlands-external-asset#top",
  "/api/batch-types/verifone-terminal-status",
  "/api/batch-types/verifone-terminal-status#contents",
  "/api/batch-types/verifone-terminal-status#example-jsonl-file",
  "/api/batch-types/verifone-terminal-status#models",
  "/api/batch-types/verifone-terminal-status#pretty-printed-example",
  "/api/batch-types/verifone-terminal-status#terminal-status",
  "/api/batch-types/verifone-terminal-status#top",
  "/api/batch-types/verifone-terminal-status#verifone-terminal-status-batch",
  "/api/batches",
  "/api/batches#batch",
  "/api/batches#batch-lifecycle",
  "/api/batches#batch-types",
  "/api/batches#batches",
  "/api/batches#contents",
  "/api/batches#create-batch-experimental",
  "/api/batches#error",
  "/api/batches#get-batch-experimental",
  "/api/batches#models",
  "/api/batches#operations",
  "/api/batches#top",
  "/api/businesses",
  "/api/businesses#business",
  "/api/businesses#businesses",
  "/api/businesses#contents",
  "/api/businesses#create-a-business-experimental",
  "/api/businesses#get-a-business-by-account-id-experimental",
  "/api/businesses#get-business-details-from-nz-company-register-experimental",
  "/api/businesses#models",
  "/api/businesses#onboarding-status-reasons",
  "/api/businesses#onboarding-statuses",
  "/api/businesses#operations",
  "/api/businesses#search-nz-company-register-experimental",
  "/api/businesses#set-business-onboarding-status",
  "/api/businesses#tax-number",
  "/api/businesses#top",
  "/api/businesses#update-a-business-experimental",
  "/api/data-types",
  "/api/data-types#bignumber",
  "/api/data-types#crn",
  "/api/data-types#data-types",
  "/api/data-types#fields",
  "/api/data-types#location",
  "/api/data-types#monetary",
  "/api/data-types#optional-fields",
  "/api/data-types#phonenumber",
  "/api/data-types#required-fields",
  "/api/data-types#timestamp",
  "/api/data-types#top",
  "/api/discrete-assets",
  "/api/external-assets",
  "/api/external-assets#contents",
  "/api/external-assets#external-assets",
  "/api/external-assets#load-an-external-asset",
  "/api/external-assets#top",
  "/api/funds-transfers",
  "/api/funds-transfers#abort-funds-transfer",
  "/api/funds-transfers#contents",
  "/api/funds-transfers#creating-a-top-up",
  "/api/funds-transfers#creating-a-withdrawal-experimental",
  "/api/funds-transfers#funds-transfers",
  "/api/funds-transfers#get-a-top-up",
  "/api/funds-transfers#get-a-withdrawal-experimental",
  "/api/funds-transfers#list-top-ups-for-an-account-experimental",
  "/api/funds-transfers#list-top-ups-for-authorized-accounts",
  "/api/funds-transfers#list-withdrawals-for-an-account-experimental",
  "/api/funds-transfers#top",
  "/api/http-status-codes",
  "/api/http-status-codes#200-ok",
  "/api/http-status-codes#400-malformed-request",
  "/api/http-status-codes#401-unauthorized",
  "/api/http-status-codes#403-forbidden",
  "/api/http-status-codes#404-route-not-found",
  "/api/http-status-codes#429-too-many-requests",
  "/api/http-status-codes#5xx-server-error",
  "/api/http-status-codes#business-rule-violated",
  "/api/http-status-codes#contents",
  "/api/http-status-codes#debugging",
  "/api/http-status-codes#debugging-1",
  "/api/http-status-codes#http-status-codes",
  "/api/http-status-codes#resource-missing-or-permission-denied",
  "/api/http-status-codes#top",
  "/api/integration-requests",
  "/api/integration-requests#activate-integration-request-experimental",
  "/api/integration-requests#configure-integration-request-experimental",
  "/api/integration-requests#contents",
  "/api/integration-requests#create-an-integration-request-experimental",
  "/api/integration-requests#delete-integration-request-experimental",
  "/api/integration-requests#get-an-integration-request-experimental",
  "/api/integration-requests#get-integration-request-configuration-experimental",
  "/api/integration-requests#integration-request",
  "/api/integration-requests#integration-requests",
  "/api/integration-requests#integration-types",
  "/api/integration-requests#models",
  "/api/integration-requests#operations",
  "/api/integration-requests#product",
  "/api/integration-requests#search-integration-requests-experimental",
  "/api/integration-requests#terminal",
  "/api/integration-requests#top",
  "/api/integrations",
  "/api/integrations#integrations",
  "/api/integrations#top",
  "/api/introduction",
  "/api/introduction#api-introduction",
  "/api/introduction#top",
  "/api/invitations",
  "/api/invitations#accept-an-invitation-experimental",
  "/api/invitations#contents",
  "/api/invitations#create-an-invitation-experimental",
  "/api/invitations#get-an-invitation-by-code-experimental",
  "/api/invitations#invitation",
  "/api/invitations#invitations",
  "/api/invitations#list-invitations-by-accountid-experimental",
  "/api/invitations#models",
  "/api/invitations#operations",
  "/api/invitations#params",
  "/api/invitations#revoke-an-invitation-experimental",
  "/api/invitations#top",
  "/api/legacy-payment-requests",
  "/api/legacy-payment-requests#cancelling-a-payment-request",
  "/api/legacy-payment-requests#contents",
  "/api/legacy-payment-requests#creating-a-payment-request",
  "/api/legacy-payment-requests#decoded-webhook-jwt-examples",
  "/api/legacy-payment-requests#getting-the-information-about-a-payment-request",
  "/api/legacy-payment-requests#legacy-payment-requests",
  "/api/legacy-payment-requests#life-cycle-events-that-trigger-webhooks",
  "/api/legacy-payment-requests#operations",
  "/api/legacy-payment-requests#paying-a-payment-request",
  "/api/legacy-payment-requests#payment-request-cancelled",
  "/api/legacy-payment-requests#payment-request-cancelled-1",
  "/api/legacy-payment-requests#payment-request-expired",
  "/api/legacy-payment-requests#refunding-a-transaction",
  "/api/legacy-payment-requests#requests-cancel",
  "/api/legacy-payment-requests#requests-create",
  "/api/legacy-payment-requests#requests-info",
  "/api/legacy-payment-requests#requests-pay",
  "/api/legacy-payment-requests#requests-void",
  "/api/legacy-payment-requests#top",
  "/api/legacy-payment-requests#transaction-completed",
  "/api/legacy-payment-requests#transaction-completed-successfully",
  "/api/legacy-payment-requests#transaction-refunded",
  "/api/legacy-payment-requests#transactions-refund",
  "/api/legacy-payment-requests#voiding-a-payment-request",
  "/api/legacy-payment-requests#webhook-jwt-validation",
  "/api/legacy-payment-requests#webhook-payload",
  "/api/legacy-payment-requests#webhook-payload-fields",
  "/api/legacy-payment-requests#webhooks",
  "/api/managed-integrations",
  "/api/managed-integrations#bank-account",
  "/api/managed-integrations#contents",
  "/api/managed-integrations#create-or-update-a-managed-integration-experimental",
  "/api/managed-integrations#get-a-managed-integration-experimental",
  "/api/managed-integrations#invitation-summary",
  "/api/managed-integrations#invitation-summary-experimental",
  "/api/managed-integrations#managed-integration",
  "/api/managed-integrations#managed-integration-1",
  "/api/managed-integrations#models",
  "/api/managed-integrations#operations",
  "/api/managed-integrations#params",
  "/api/managed-integrations#paypal-referral",
  "/api/managed-integrations#top",
  "/api/media-uploads",
  "/api/media-uploads#contents",
  "/api/media-uploads#create-a-presigned-url-for-media-upload-experimental",
  "/api/media-uploads#get-media-upload-location-experimental",
  "/api/media-uploads#media-upload",
  "/api/media-uploads#media-uploads",
  "/api/media-uploads#model",
  "/api/media-uploads#models",
  "/api/media-uploads#operations",
  "/api/media-uploads#properties",
  "/api/media-uploads#top",
  "/api/merchant-configs",
  "/api/merchant-configs#contents",
  "/api/merchant-configs#create-a-merchant-config",
  "/api/merchant-configs#get-a-merchant-config",
  "/api/merchant-configs#list-merchant-configs",
  "/api/merchant-configs#merchant-config",
  "/api/merchant-configs#merchant-configs",
  "/api/merchant-configs#models",
  "/api/merchant-configs#operations",
  "/api/merchant-configs#payment-option-config",
  "/api/merchant-configs#top",
  "/api/merchant-configs#update-a-merchant-config",
  "/api/merchants",
  "/api/merchants#accepted-asset",
  "/api/merchants#contents",
  "/api/merchants#create-a-merchant",
  "/api/merchants#get-a-merchant",
  "/api/merchants#list-all-merchants-deprecated",
  "/api/merchants#list-merchants-for-account",
  "/api/merchants#merchant",
  "/api/merchants#merchant-search-result",
  "/api/merchants#merchants",
  "/api/merchants#models",
  "/api/merchants#onboarding-status-reasons",
  "/api/merchants#onboarding-statuses",
  "/api/merchants#operations",
  "/api/merchants#product",
  "/api/merchants#search-merchants-experimental",
  "/api/merchants#set-merchant-onboarding-status",
  "/api/merchants#settlement-config",
  "/api/merchants#top",
  "/api/merchants#update-a-merchant",
  "/api/pagination",
  "/api/pagination#example",
  "/api/pagination#example-1",
  "/api/pagination#models",
  "/api/pagination#optional-fields",
  "/api/pagination#optional-fields-1",
  "/api/pagination#pagination",
  "/api/pagination#request",
  "/api/pagination#required-fields",
  "/api/pagination#response",
  "/api/pagination#top",
  "/api/patron-codes",
  "/api/patron-codes#contents",
  "/api/patron-codes#creating-a-patron-code",
  "/api/patron-codes#models",
  "/api/patron-codes#operations",
  "/api/patron-codes#patron-code",
  "/api/patron-codes#patron-codes",
  "/api/patron-codes#retrieving-a-patron-code",
  "/api/patron-codes#retrieving-a-patron-code-by-barcode",
  "/api/patron-codes#test-scenario-name",
  "/api/patron-codes#top",
  "/api/payment-activities",
  "/api/payment-requests",
  "/api/payment-requests#accept-payment-condition",
  "/api/payment-requests#accept-payment-condition-for-a-payment-request",
  "/api/payment-requests#accepted-collection",
  "/api/payment-requests#asset-total",
  "/api/payment-requests#cancellation-reasons",
  "/api/payment-requests#contents",
  "/api/payment-requests#create-a-payment-request",
  "/api/payment-requests#decline-payment-condition-for-a-payment-request",
  "/api/payment-requests#get-a-payment-request",
  "/api/payment-requests#get-a-payment-request-by-short-code",
  "/api/payment-requests#get-a-payment-request-linked-to-a-patron-code",
  "/api/payment-requests#line-item",
  "/api/payment-requests#list-payment-activities-for-a-merchant",
  "/api/payment-requests#list-payment-activities-for-a-payment-request",
  "/api/payment-requests#make-a-confirmation-against-a-pre-auth-payment-request",
  "/api/payment-requests#models",
  "/api/payment-requests#operations",
  "/api/payment-requests#paid-by",
  "/api/payment-requests#pay",
  "/api/payment-requests#pay-a-payment-request",
  "/api/payment-requests#payment-activity",
  "/api/payment-requests#payment-condition",
  "/api/payment-requests#payment-option",
  "/api/payment-requests#payment-request",
  "/api/payment-requests#payment-requests",
  "/api/payment-requests#product-classification",
  "/api/payment-requests#refund",
  "/api/payment-requests#refund-a-payment-request",
  "/api/payment-requests#release-funds-held-for-a-pre-auth-payment-request",
  "/api/payment-requests#top",
  "/api/payment-requests#void-a-payment-request",
  "/api/profiles",
  "/api/profiles#contents",
  "/api/profiles#get-profile-experimental",
  "/api/profiles#models",
  "/api/profiles#operations",
  "/api/profiles#profile",
  "/api/profiles#profile-1",
  "/api/profiles#top",
  "/api/profiles#update-a-profile-experimental",
  "/api/quotas",
  "/api/quotas#contents",
  "/api/quotas#getting-account-quotas-experimental",
  "/api/quotas#models",
  "/api/quotas#operations",
  "/api/quotas#quota",
  "/api/quotas#quotas",
  "/api/quotas#top",
  "/api/scanned-codes",
  "/api/scanned-codes#contents",
  "/api/scanned-codes#decode-scanned-code",
  "/api/scanned-codes#models",
  "/api/scanned-codes#operations",
  "/api/scanned-codes#scanned-code",
  "/api/scanned-codes#scanned-codes",
  "/api/scanned-codes#top",
  "/api/section/accounts",
  "/api/section/assets",
  "/api/section/bank-accounts",
  "/api/section/batches",
  "/api/section/integrations",
  "/api/section/merchants",
  "/api/section/payment-requests",
  "/api/section/scanned-codes",
  "/api/section/settlements",
  "/api/settlements",
  "/api/settlements#contents",
  "/api/settlements#list-settlements",
  "/api/settlements#models",
  "/api/settlements#operations",
  "/api/settlements#settlement",
  "/api/settlements#settlements",
  "/api/settlements#top",
  "/api/tokens",
  "/api/tokens#allowed-products-experimental",
  "/api/tokens#allowedProducts",
  "/api/tokens#contents",
  "/api/tokens#create-redemption-condition-experimental",
  "/api/tokens#create-token-collection-experimental",
  "/api/tokens#create-token-experimental",
  "/api/tokens#list-token-collections-for-account-experimental",
  "/api/tokens#models",
  "/api/tokens#operations",
  "/api/tokens#redemption-condition-experimental",
  "/api/tokens#token-collection",
  "/api/tokens#token-collection-experimental",
  "/api/tokens#token-expires-after-experimental",
  "/api/tokens#tokenExpiresAfter",
  "/api/tokens#tokens",
  "/api/tokens#top",
  "/api/transacting",
  "/api/wallets",
  "/api/wallets#contents",
  "/api/wallets#creating-a-wallet",
  "/api/wallets#listing-authorized-wallets",
  "/api/wallets#listing-wallet-transactions-experimental",
  "/api/wallets#settlement-wallets",
  "/api/wallets#settlement-wallets-experimental",
  "/api/wallets#top",
  "/api/wallets#wallets",
  "/assets",
  "/assets/asset-categories",
  "/assets/asset-transfers",
  "/assets/discrete-assets",
  "/assets/wallets",
  "/auth",
  "/connections/farmlands",
  "/fiat",
  "/fiat/authorities",
  "/fiat/bank-accounts",
  "/fiat/bank-authorities",
  "/fiat/funds-transfers",
  "/guides/creating-digital-tokens",
  "/guides/creating-digital-tokens#implementation",
  "/guides/creating-digital-tokens#redemption-conditions",
  "/guides/creating-digital-tokens#token-collection",
  "/guides/creating-test-money",
  "/guides/creating-test-money#via-api",
  "/guides/creating-test-money#via-centrapay-app",
  "/guides/ecommerce-website",
  "/guides/example-oidc-consumer",
  "/guides/example-oidc-consumer#detect-oidc-user-refresh",
  "/guides/example-oidc-consumer#minimal-oidc-client-usage",
  "/guides/example-oidc-consumer#oidc-client-logout",
  "/guides/example-oidc-consumer#oidc-user-manager-configuration",
  "/guides/example-oidc-consumer#trigger-token-refresh",
  "/guides/farmlands-card-partner-support",
  "/guides/farmlands-card-partner-support#p1---severe-business-disruption",
  "/guides/farmlands-card-partner-support#p2---major-business-disruption",
  "/guides/farmlands-card-partner-support#p3---minor-business-disruption",
  "/guides/farmlands-card-partner-support#p4---adhoc-requests-as-requested-by-farmlands",
  "/guides/farmlands-card-partner-support#sla-support-level",
  "/guides/farmlands-card-partner-support#support-for-portal-users",
  "/guides/farmlands-card-partner-support#support-for-pos-integrators",
  "/guides/farmlands-card-partner-support#trouble-shooting",
  "/guides/farmlands-portal",
  "/guides/farmlands-portal#authorisation-history",
  "/guides/farmlands-portal#authorisation-responses",
  "/guides/farmlands-portal#bank-accounts",
  "/guides/farmlands-portal#business-information",
  "/guides/farmlands-portal#create-a-farmlands-authorisation",
  "/guides/farmlands-portal#dashboard",
  "/guides/farmlands-portal#faqs",
  "/guides/farmlands-portal#farmlands-authorisations",
  "/guides/farmlands-portal#getting-started",
  "/guides/farmlands-portal#invoicing--settlement",
  "/guides/farmlands-portal#log-in",
  "/guides/farmlands-portal#manage-my-centrapay-business-account",
  "/guides/farmlands-portal#members",
  "/guides/farmlands-portal#merchants",
  "/guides/farmlands-portal#registration",
  "/guides/farmlands-portal#release-hold-on-customer-funds",
  "/guides/farmlands-portal#releasing-funds",
  "/guides/farmlands-portal#setting-a-default-merchant",
  "/guides/farmlands-portal#settings",
  "/guides/farmlands-portal#support",
  "/guides/farmlands-pos-integration",
  "/guides/farmlands-pos-integration#authentication",
  "/guides/farmlands-pos-integration#basic-requirements",
  "/guides/farmlands-pos-integration#cardholder-not-present",
  "/guides/farmlands-pos-integration#centrapay-integration-benefits",
  "/guides/farmlands-pos-integration#certification-requirements",
  "/guides/farmlands-pos-integration#handling-payment-errors",
  "/guides/farmlands-pos-integration#merchant-configuration",
  "/guides/farmlands-pos-integration#next-steps",
  "/guides/farmlands-pos-integration#payment-conditions",
  "/guides/farmlands-pos-integration#payment-flow-implementation",
  "/guides/farmlands-pos-integration#payment-flow-overview",
  "/guides/farmlands-pos-integration#pre-auth",
  "/guides/farmlands-pos-integration#refunds",
  "/guides/farmlands-pos-integration#short-codes",
  "/guides/farmlands-pos-integration#testing-your-integration",
  "/guides/farmlands-pos-integration#validating-farmlands-barcodes",
  "/guides/initiating-refunds",
  "/guides/initiating-refunds#by-short-code",
  "/guides/initiating-refunds#by-transaction-reference",
  "/guides/initiating-refunds#multi-asset-selection",
  "/guides/initiating-refunds#obtaining-a-payment-request-id",
  "/guides/initiating-refunds#refund-error-handling",
  "/guides/initiating-refunds#refund-idempotency",
  "/guides/integrating-third-party-asset",
  "/guides/integrating-third-party-asset#authentication",
  "/guides/integrating-third-party-asset#cancel-endpoint",
  "/guides/integrating-third-party-asset#contact-details",
  "/guides/integrating-third-party-asset#defining-a-payment-method",
  "/guides/integrating-third-party-asset#description",
  "/guides/integrating-third-party-asset#error-codes",
  "/guides/integrating-third-party-asset#example-definition",
  "/guides/integrating-third-party-asset#failure-reasons",
  "/guides/integrating-third-party-asset#get-transaction-endpoint",
  "/guides/integrating-third-party-asset#implementation-checklist",
  "/guides/integrating-third-party-asset#liveness",
  "/guides/integrating-third-party-asset#namespace",
  "/guides/integrating-third-party-asset#pay-endpoint",
  "/guides/integrating-third-party-asset#protocol",
  "/guides/integrating-third-party-asset#refund-endpoint",
  "/guides/integrating-third-party-asset#settlement",
  "/guides/integrating-third-party-asset#statuses",
  "/guides/integrating-third-party-asset#supported-currencies",
  "/guides/integrating-third-party-asset#transaction-attempt-model",
  "/guides/integrating-third-party-asset#transaction-idempotency",
  "/guides/integrating-third-party-asset#uplink-api-requirements",
  "/guides/integrating-third-party-asset#uplink-api-spec",
  "/guides/integrating-third-party-asset#url",
  "/guides/line-items",
  "/guides/line-items#determining-paid-for-line-items",
  "/guides/line-items#restrictions",
  "/guides/loading-and-sending-assets",
  "/guides/loading-and-sending-assets#example-bulk-distribution-of-giftcards",
  "/guides/loading-and-sending-assets#loading-giftcards",
  "/guides/loading-and-sending-assets#sending-assets",
  "/guides/merchant-integration-barcode-flow",
  "/guides/merchant-integration-barcode-flow#barcode-flow",
  "/guides/merchant-integration-barcode-flow#checking-barcode-details",
  "/guides/merchant-integration-barcode-flow#quick-pay-flow",
  "/guides/merchant-integration-error-handling",
  "/guides/merchant-integration-error-handling#configure-pos-timeout",
  "/guides/merchant-integration-error-handling#resolving-persistent-errors",
  "/guides/merchant-integration-error-handling#respect-payment-status",
  "/guides/merchant-integration-error-handling#retry-unknown-errors",
  "/guides/merchant-integration-error-handling#void-unknown-status",
  "/guides/merchant-integration-qr-code-flow",
  "/guides/merchant-terminals",
  "/guides/partial-payment-extension",
  "/guides/partial-payment-extension#creating-the-payment-request",
  "/guides/partial-payment-extension#hybrid-partial-payment-scenario",
  "/guides/partial-payment-extension#implementation",
  "/guides/partial-payment-extension#merchant-driven-scenario",
  "/guides/partial-payment-extension#partial-payment-flow",
  "/guides/partial-payment-extension#patron-driven-scenario",
  "/guides/partial-payment-extension#polling-for-payment-confirmation",
  "/guides/partial-payment-extension#see-also",
  "/guides/patron-not-present",
  "/guides/payment-conditions",
  "/guides/payment-conditions#additional-behaviors",
  "/guides/payment-conditions#implementation",
  "/guides/payment-flows",
  "/guides/payment-flows#dynamic-merchant-qr-code",
  "/guides/payment-flows#dynamic-patron-barcode",
  "/guides/payment-flows#merchant-presented",
  "/guides/payment-flows#patron-presented",
  "/guides/payment-flows#quick-pay",
  "/guides/payment-flows#static-merchant-qr-code",
  "/guides/payment-flows#static-patron-barcode",
  "/guides/payment-terminals",
  "/guides/payment-terminals#api-keys",
  "/guides/payment-terminals#displaying-a-button",
  "/guides/payment-terminals#displaying-an-icon",
  "/guides/payment-terminals#displaying-qr-codes",
  "/guides/payment-terminals#example-flows",
  "/guides/payment-terminals#integration-architecture",
  "/guides/payment-terminals#merchant-configs",
  "/guides/payment-terminals#mitigating-network-issues",
  "/guides/payment-terminals#terminal-interface-guidelines",
  "/guides/point-of-sale",
  "/guides/point-of-sale#contact-us",
  "/guides/point-of-sale#core-requirements",
  "/guides/point-of-sale#optional-protocol-extensions",
  "/guides/requesting-payment",
  "/guides/requesting-payment#connecting-with-patrons",
  "/guides/requesting-payment#handling-payment-errors",
  "/guides/requesting-payment#polling-for-payment-confirmation",
  "/guides/requesting-payment#protocol-extensions",
  "/guides/requesting-payment#required-fields",
  "/guides/requesting-payment#short-codes",
  "/guides/requesting-pre-auth",
  "/guides/requesting-pre-auth#authorize",
  "/guides/requesting-pre-auth#confirm",
  "/guides/requesting-pre-auth#expiry",
  "/guides/requesting-pre-auth#pre-auth-flow",
  "/guides/requesting-pre-auth#refund",
  "/guides/requesting-pre-auth#release",
  "/guides/requesting-pre-auth#restrictions",
  "/guides/requesting-pre-auth#void",
  "/guides/third-party-application-payments",
  "/guides/third-party-application-payments#authenticating-api-calls-on-behalf-of-users",
  "/guides/third-party-application-payments#get-the-payment-request",
  "/guides/third-party-application-payments#overview",
  "/guides/third-party-application-payments#partial-pay",
  "/guides/third-party-application-payments#pay-the-payment-request",
  "/guides/third-party-application-payments#payment-extensions",
  "/guides/third-party-application-payments#payment-flow-implementation",
  "/guides/third-party-application-payments#payment-flow-overview",
  "/guides/third-party-application-payments#scan-centrapay-qr-code",
  "/guides/third-party-application-payments#testing",
  "/guides/transaction-reporting",
  "/merchants",
  "/profile",
  "/quotas",
  "/settlements",
  "/transacting",
]
`;
